domains/Discussions/Database/Migrations/2020_10_19_175943_create_question_answers_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Domains\Discussions\Models\Question;
+use Domains\Accounts\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateQuestionAnswersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('question_answers', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(User::class, 'author_id');
+            $table->foreignIdFor(Question::class, 'question_id');
+            $table->text('content');
+            $table->timestampsTz();
```

Comment:
I think we should index the created_at timestamp column, too, since #32 will require the questions to be ordered from the latest to the oldest.

Reply:
ok.